---

uint8 exception_code
uint16 value